plugins/islandora_context_reaction_restrict_by_ip.inc

```diff
@@ -111,7 +111,7 @@ class IslandoraContextReactionRestrictByIp extends context_reaction {
       if (!strlen($range)) {
         continue;
       }
-      list($low, $high) = explode(':', $range);
+      list($low, $high) = array_pad(explode(':', $range, 2), 2, NULL);
```

Comment:
Thank you!  After working with the plugin a little more, I've come to like the consistency of using the IP range notation even for single IP addresses.   Maybe create a question ticket in case others feel strongly about not using the range notation for single IP addresses?